src/analysis/network/qgslinevectorlayerdirector.h

```diff
@@ -37,20 +32,35 @@ class ANALYSIS_EXPORT QgsLineVectorLayerDirector : public QgsGraphDirector
 
   public:
 
+    /** Road direction
+     * Road can be one-way with direct flow (one can move only from the start
+     * point to the end point), one-way with reversed flow (one can move only
+     * from the end point to the start point) and bidirectional or two-way
+     * (one can move in any direction)
+     */
+    enum RoadDirection
```

Comment:
Actually scratch that - I think "road" should not be used here at all (since the network could be pipes, wires,... ). I'd go with something like

    enum Direction
    {
        DirectionForward,
        DirectionBackward,
        DirectionBoth,
     }

and also go for removing the mentions of "road" in the constructor doxygen tags. This relates to http://hub.qgis.org/issues/12158